packages/math-input/src/components/input/math-input.js

```diff
@@ -836,7 +900,7 @@ class MathInput extends React.Component {
                 onTouchStart={this.handleTouchStart}
                 onTouchMove={this.handleTouchMove}
                 onTouchEnd={this.handleTouchEnd}
-                onClick={(e) => e.stopPropagation()}
+                onClick={(e: $FlowFixMe) => e.stopPropagation()}
```

Comment:
Cool.  One thing that I think we could look into changing is replacing the `<View>` and `<Text>` components that `math-input` has (in the `fake-react-native-web` folder) with WB equivalents.  

No action required.